Composer/packages/lib/shared/src/constant.ts

```diff
@@ -10,3 +11,5 @@ export const FieldNames = {
   DefaultCase: 'default',
   Cases: 'cases',
 };
+
+export const thisPC = formatMessage('This PC');
```

Comment:
This can't be statically export using `formatMessage`. You should see this in the browser console as a warning. Any usage of `formatMessage` must be at runtime because it uses runtime settings for determining the locale.

Reply:
Have you considered using `/` instead of `This PC`? I'm concerned that by translating this string at runtime, there may be some hidden bugs due to the translation. Also, the server is not configured for i18n at this point, so it would not get translated on the server.

Reply:
yes, i was also thinking about "/" don't require any extra explanation

Reply:
switched to /.